app/services/search/query_parser.rb

```diff
@@ -0,0 +1,21 @@
+module Search
+  class QueryParser < Parslet::Parser
+    rule(:whitespace) { match['\s'].repeat(1) }
+    rule(:whitespace?) { whitespace.maybe }
+
+    rule(:quote) { str('"') >> whitespace? }
+    rule(:quote?) { quote.maybe }
+
```

Comment:
should we have both a `:single_quote?` and a `:double_quote?` rule??

Reply:
Probably not - google just does the exact match on double quotes doesn't it. Plus then you'd have issue with matching on things like HRMC's.

Reply:
I followed [Google's lead](https://support.google.com/websearch/answer/2466433):

> **Search for an exact match**
> Put a word or phrase inside quotes. For example, "tallest building".